<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainForm</class>
 <widget class="QWidget" name="MainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>147</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>147</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>147</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Neuromodel</string>
  </property>
  <widget class="QPushButton" name="StartStopButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>85</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Start / Stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SettingsButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>104</y>
     <width>85</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>App Settings</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="NoizeGroupBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>33</y>
     <width>90</width>
     <height>80</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="title">
    <string>Noize</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QComboBox" name="NoizeTypeComboBox">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>25</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="modelColumn">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>Relative</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fixed</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="NoizeValueSpinBox">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>53</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="NeuronParametersButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>48</y>
     <width>85</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Parameters</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NetworkButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>76</y>
     <width>85</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Network</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
